packages/client/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@sendgrid/client",
   "description": "Twilio SendGrid NodeJS API client",
-  "version": "7.4.0",
+  "version": "7.4.1",
```

Comment:
Please revert this version bump as well.

Reply:
Reverted.  Also reverted the reference to this version in `mail/package.json`.